### ISSUE TITLE
server refuses addon stb_type/ver

### ISSUE THREAD

**Issue by kotc** [kodi-pvr/pvr.stalker#192]
as in title, if i only put hostname or ip as a server address it tries to connect, but obviously fails on redirect. i see it in apache/nginx logs. but my portal is at custom path, for example http://someip/custom_path/c/, if i try to put the path as a server address it doesnt even try to connect (no entries in apache/nginx logs).

version: 19.0.3.1

**Comment by phunkyfish** [comment]
Please add a full Debug log

**Comment by kotc** [comment]
how can i get it?

**Comment by kotc** [comment]
nvm, server entry is parsed ok, but the problem is bad stb_type/ver sent, is it possible to modify it or it's unsupported?

**Comment by kotc** [comment]
for other people to save them from googling:
you can ssh onto your kodi box and check location depending on the distro:
https://kodi.wiki/view/Log_file
in my case libreelec:
tail -f /storage/.kodi/temp/kodi.log |grep -i stalker

**Comment by phunkyfish** [comment]
> nvm, server entry is parsed ok, but the problem is bad stb_type/ver sent, is it possible to modify it or it's unsupported?

Why is it bad? Or does the add-on only support certain versions?

**Comment by kotc** [comment]
bad because server doesnt allow 'mag250' and probably 'old' firmware version that is sent

**Comment by kotc** [comment]
2022-09-24 14:57:44.410 T:3308    ERROR <general>: AddOnLog: pvr.stalker: GetProfile: status=1 | msg=Old firmware, missing metrics or hash | block_msg=Your device is not supported.

**Comment by phunkyfish** [comment]
So you need to upgrade the firmware on your device?

**Comment by kotc** [comment]
no, i plan to recompile addon to send proper version string (from mag254) and model type, so the server doesnt act up unnecessarily

**Comment by phunkyfish** [comment]
If you get it working please PR the change here so other users can benefit from it.
